tools/metrics/Makefile

```diff
@@ -10,15 +10,17 @@ endif
 
 .PHONY: acceptance-tests
 acceptance-tests:
-	ginkgo acceptance
+	go run github.com/onsi/ginkgo/v2/ginkgo acceptance
 
 .PHONY: pkg-tests
+# This looks broken. i can't see how this ever ran. should we remove it?
+# (`. "github.com/onsi/ginkgo/v2"` is missing???)
```

Comment:
Is this not added by your paas-billing PR?

Reply:
the comment is, the test is not. i can't see how `make pkg-tests` ever ran correctly, so I dropped a comment in to draw attention to it in future / make it so the next person doesn't end up as confused as I was when I couldn't get these tests to run

Reply:
Yeah ok noticed this is on the paas-metrics project I see.